autoload/lsp/completion.vim

```diff
@@ -310,6 +310,26 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
     endif  
 
     d.user_data = item
+
+    # Condense completion menu items to single words (plus kind)
+    # Move all additional details to the info popup
+    # Caveat: LazyDoc will override moved details!
+    if lspOpts.condensedCompletionMenu
+      const SEP = (s) => empty(s) ? "" : "\n- - -\n"
+      var infoText = ''
+      if d->has_key('abbr') && len(d.abbr) > len(d.word)
+        infoText ..= SEP(infoText) .. '    ' .. d.abbr
+        d.abbr = d.word
```

Comment:
why change this?

Reply:
In order to make the completion menu as narrow as possible the shorter text of __abbr__ or __word__ is used. When __abbr__ is exchanged by __word__ (because __abbr__ is to verbose) then __abbr__ will be moved into the info-popup.

Reply:
i means d.word _maybe_ empty; 
and changing those value not sure had outage.

Reply:
No ,`d.word` cannot be empty.
This is ensured by the specification of the language server protocol that mandates that at least `CompletionItem.label` must be not empty. And the _fallback_  (which might be in my eyes not straightforward regarding the handling of fuzzy-matching, see also #584 ) of this code (beginning with line 197) ensures this:
```
    if item->has_key('textEdit') &&
	lspOpts.completionMatcherValue != opt.COMPLETIONMATCHER_FUZZY
...
    elseif item->has_key('insertText')
...
    else
      d.word = item.label
    endif
```

`